pkg/agent/config.go

```diff
@@ -163,8 +164,6 @@ type Config struct {
 	// PCAFilters set the filters to determine packets to filter using Packet Capture Agent (PCA). It is a comma separated set.
 	// The format is [protocol], [port number] Example: PCA_FILTER = "tcp,80". Currently, we support 'tcp','udp','sctp' for protocol.
 	PCAFilters string `env:"PCA_FILTER"`
-	// PCAServerPort is the port PCA Server starts at, when ENABLE_PCA variable is set to true.
-	PCAServerPort int `env:"PCA_SERVER_PORT" envDefault:"9990"`
```

Comment:
I remember we had this discussion with Shachee previously, that reusing the `FLOWS_TARGET_PORT` env for pcap was misleading, and for two reasons: first because it has "FLOWS" in the name, and second because in the case of the TCP impl, if I remember correctly, this isn't a target port (ie. a remote host port) but it's a server port, uses to listen for clients who want to poll data. So that's why we ended up with this new env.

Reply:
FYI I addressed this in that way in:
https://github.com/netobserv/netobserv-ebpf-agent/pull/291/commits/8cbecafbadef42631196dd60e49b7e5547464859
https://github.com/netobserv/netobserv-ebpf-agent/pull/291/commits/3e86b43ccc3c9fbb2d89804537b5110b002a8ba0

Reply:
idk it still bother me to mix server and client config into a single configuration ; you kept "target" which IMO is still misleading for a server configuration, it really looks like we are configuring a client. idk if it's just me, @msherif1234 what do you think? I defer to you :-)

Reply:
well if we get rid of pcap w/ tcp , hence we have no server mode anyway, then I guess it answers the question

Reply:
As mention above I changed the config to get rid of these mentions. That's more generic for future implementations.
On top of that I removed tcp export for packet capture.

Let me know if the current state match your expectations

Reply:
Well, sorry but, now that it's clearly only a client connection there's no reason to remove "target" from the names? It was inconsistent if we were mixing client & server, but now that we don't, it's fine and more precise than just "host" / "port"
Anyway I don't want to block this PR indefinitely, we can do that later